m/mesa/stone.yaml

```diff
@@ -103,15 +109,15 @@ setup       : |
         -Dgles1=disabled \
         -Dglvnd=enabled \
         -Dllvm=enabled \
+        -Dmesa-clc=system \
         -Dosmesa=true \
         -Dzstd=enabled \
         -Dplatforms=x11,wayland \
         -Dshared-llvm=enabled \
         -Dvideo-codecs=all \
         -Dmicrosoft-clc=disabled \
-        -Dintel-clc=system \
-        -Dvulkan-drivers=amd,intel,intel_hasvk,swrast,virtio$EXTRA_VK_DRIVERS \
-        -Dvulkan-layers=device-select,overlay,screenshot
+        -Dvulkan-drivers=amd,gfxstream,intel,intel_hasvk,swrast,virtio$EXTRA_VK_DRIVERS \
```

Comment:
As discussed on Matrix remove gfxstream from here and move it to the `EXTRA_VK_DRIVERS` section on line 87.

Reply:
Ok, adjusted.